doc/rst/quick.rst

```diff
@@ -19,21 +19,22 @@ Each step is explained in detail in the following sections.
 
 #. If using a Windows host computer, download the `USB Audio Class 2.0 Evaluation Driver for Windows`.
 
-   * On the `XMOS website USB & Multichannel Audio page <http://www.xmos.com/develop/usb-multichannel-audio>`__, follow the `DRIVER SUPPORT` link, and click on `Download`. Once downloaded, run the executable and install the driver.
+   * On the `XMOS USB & Multichannel Audio developer webpage <http://www.xmos.com/develop/usb-multichannel-audio>`__, follow the `DRIVER SUPPORT` link, and click on `DOWNLOAD EVALUATION DRIVER`. Once downloaded, run the executable and install the driver.
 
    (Section :ref:`quick_start_windows_driver`.)
 
 #. Download and install the the `XMOS XTC Tools <http://www.xmos.com/software-tools>`_
 
-   * The minimum required XTC Tools version for compiling USB Audio applications can be found in the README. Make sure to download the correct version of the tools.
+   * The required XTC Tools version for compiling USB Audio applications can be found in the README file supplied in the software package.
+     Be sure to download the correct version of the tools.
 
    (Section :ref:`quick_start_tools`)
 
-#. Compile the firmware relavant to the available reference hardware platform .
+#. Compile the software relavant to the available board.
 
    (Section: :ref:`quick_start_building`)
 
-#. Connect the board to the development system using (using the supplied xTAG if required), and program the firmware onto the board.
+#. Connect the board to the host machine (using the supplied xTAG if required), and program the compiled binary onto the board.
```

Comment:
Maybe say there are two connections - Host connection to USB and debug connection which may be either integrated on  the board or via an external xTAG, depending on board.

Reply:
Done

Reply:
Actually, looking at the supported boards, XTAG is now dead.